PyInstaller/loader/rthooks/pyi_rth_qt5plugins.py

```diff
@@ -29,6 +29,13 @@
     os.environ['QT_PLUGIN_PATH'] = ''
     del os.environ['QT_PLUGIN_PATH']
 
+# The packaging system names sip PyQt5.sip, so make it available as "sip"
+try:
```

Comment:
We should first test if `sip` can be imported:
```python
try:
  import sip
except ImportError:
  try:
    ... your code ... 
```
It would be even better if we'd verify the module name.